client/go/cmd/curl.go

```diff
@@ -60,8 +57,36 @@ $ vespa curl -- -v --data-urlencode "yql=select * from music where album contain
 		if err != nil {
 			return err
 		}
-		c.PrivateKey = privateKeyFile
-		c.Certificate = certificateFile
+		switch curlService {
+		case "deploy":
+			if !vespa.Auth0AccessTokenEnabled() {
```

Comment:
A test should be added for this branch.

Reply:
Added a test for the "target is local" case, would have been nice to have something on the Auth0-part as well, but I see no way to do that without mocking away the entire Auth0-flow, which will make the test useless.